src/display/editor/stamp.js

```diff
@@ -536,6 +545,10 @@ class StampEditor extends AnnotationEditor {
       }
     });
     this.#observer.observe(this.div);
+    this._uiManager._signal.addEventListener("abort", () => {
+      this.#observer.disconnect();
```

Comment:
Also, is there a risk that this could have been removed already in the `remove` method?
```suggestion
      this.#observer?.disconnect();
```